translations/monero-core_de.ts

```diff
@@ -815,17 +815,17 @@
     <message>
         <location filename="../pages/Receive.qml" line="385"/>
         <source>Save QrCode</source>
-        <translation type="unfinished"></translation>
+        <translation>QR-Code speichern</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="386"/>
         <source>Failed to save QrCode to </source>
-        <translation type="unfinished"></translation>
+        <translation>Konnte QR-Code nicht speichern als </translation>
```

Comment:
I think its used for saving the QR-Code using the right mouse button, so "unter" instead of "als"